docs/docs/integrations/vectorstores/elasticsearch.ipynb

```diff
@@ -736,6 +736,50 @@
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "05cdb43d-5e46-46f6-a2dc-91df4aa56ec7",
+   "metadata": {},
+   "source": [
+    "## BM25RetrievalStrategy\n",
+    "This strategy allows the user to perform searches using pure BM25 without vector search.\n",
+    "\n",
+    "To use this, specify `BM25RetrievalStrategy` in `ElasticsearchStore` constructor.\n",
+    "\n",
+    "Note that in the example below, the embedding option is not specified, indicating that the search is conducted without using embeddings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4464a657-08c5-4a1a-b0e8-dba65f5b7ec0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[Document(page_content='foo'), Document(page_content='foo bar'), Document(page_content='foo bar baz')]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from langchain_elasticsearch.vectorstores import ElasticsearchStore\n",
```

Comment:
```suggestion
    "from langchain_elasticsearch import ElasticsearchStore\n",
```

sorry, I caught this after my approval

Reply:
Thanks for your suggestion! I applied that and confirmed it works on the notebook.